libs/coin-evm/docs/broadcast.md

```diff
@@ -0,0 +1,8 @@
+# `broadcast.ts`
+
+## Methods
+
+#### broadcast <sub><sup><sub><sup>[standard]</sup><sub></sup><sub>
+In charge of sending a signed transaction to a node in order for that node to add it into its [mempool](https://www.alchemy.com/overviews/what-is-a-mempool). 
+Once received, the node should return the transaction *hash* that will represent that transaction on all the nodes & explorers of the network. This hash will then be used to patch the *optimisticOperation* created by the live, helping the process of overriding that *optimisticOperation* once the finalized operation is received after a new synchronization.
```

Comment:
`the node should return the transaction *hash* that will represent that transaction`

It returns an operation with updated hash, rest is :+1:

Reply:
Nope, the node itself will return a transaction hash: https://docs.infura.io/networks/ethereum/json-rpc-methods/eth_sendrawtransaction#response
That hash is then used to update the optimistic operation and that operation is then returned by the broadcast method.

Reply:
Oops, indeed we are talking about the node itself not the method